library/core/src/mem/mod.rs

```diff
@@ -1326,7 +1326,7 @@ impl<T> SizedTypeProperties for T {}
 /// # Examples
 ///
 /// ```
-/// #![feature(offset_of_enum, offset_of_nested)]
+/// #![feature(offset_of_enum)]
```

Comment:
Doc tests are built using the bootstrap compiler, so this needs a `#![cfg_attr(bootstrap, feature(offset_of_nested))]`.